server/src/test/java/org/elasticsearch/health/HealthServiceTests.java

```diff
@@ -50,6 +50,32 @@ public void testShouldReturnGroupedIndicators() {
         );
     }
 
+    public void testDuplicateIndicatorNamess() {
```

Comment:
should this only test `createComponentFromIndicators` (which can be default protected) ?

Reply:
Yeah I can change it to do that. Unless it's really difficult to cover the code, I usually try to go through the public API for testing.